makefile

```diff
@@ -0,0 +1,22 @@
+# Create a blank makefile and setup a default: task as shown in the lecture demo, so it just cats the makefile itself
+default:
```

Comment:
This format works, however, just for readability can we add a space above the job names?  Or rather above the comment above the job names... you know what I mean right?  just to separate the ideas in the makefile a little. Thanks!